bin/search.js

```diff
@@ -51,13 +56,15 @@ function typeSearch(el) {
             }
         }
     }
+    var query = "";
     function createDataSource() {
         var local = JSON.parse(window.localStorage.getItem(localStorageDataKey)) || undefined;
         var bh = new Bloodhound({
             datumTokenizer: function (entry) {
                 return [entry.l, entry.p, entry.t].concat(entry.g).concat(entry.m);

```

Comment:
But that won't solve the ranking problem would it? Ideally we should put 'angularjs' on top when one searches for 'angular'.